packages/firestore/rollup.config.exp.js

```diff
@@ -42,7 +43,8 @@ const nodePlugins = [
       }
     },
     clean: true,
-    transformers: removeAssertTransformer
+    abortOnError: false,
+    transformers: [...removeAssertTransformer, importPathTransformer]
```

Comment:
Can this and `removeAssertAndPrefixInternalTransformer` be plural ("transformers") so the spread makes sense?